bool autotrim
float64 heading
float64 angle_of_attack

